integreat_cms/locale/de/LC_MESSAGES/django.po

```diff
@@ -7910,6 +7910,11 @@ msgstr "PDF-Datei konnte nicht erfolgreich erzeugt werden."
 msgid "Cannot generate slug from %(fallback)s."
 msgstr "Slug kann nicht aus dem %(fallback)s abgeleitet werden."
 
+#: cms/utils/slug_utils.py
+#, python-format
+msgid "%(slug)s is a reserved slug."
+msgstr "%(slug)s ist ein reservierter URL-Parameter"
+
```

Comment:
```suggestion
```

Reply:
Why remove these lines? :)

Reply:
Just because I saw that the same translation (just with the `{}` formatting) was already included in the translation file (commented out) and I thought we don't need to keep both. I intended to keep one of them, but now I saw you removed both of them, that's no problem, no need to re-add it :sweat_smile: